qt/scientific_interfaces/Indirect/IndirectFitAnalysisTab.cpp

```diff
@@ -813,8 +813,7 @@ void IndirectFitAnalysisTab::respondToBackgroundChanged(double value) {
 
 void IndirectFitAnalysisTab::respondToFunctionChanged() {
   setModelFitFunction();
-  m_plotPresenter->updatePlots();
-  m_plotPresenter->updateGuessAvailability();
+  m_plotPresenter->updateFit();
```

Comment:
I think there is a slight change in functionality here. If you do a fit and then change the fit function the calculated fit line will remain unchanged whilst the plot guess will be updated. Don't know whether this is an issue or not so may be worth checking with scientists.

Reply:
@StephenSmith25 @Mantid-Matthew If the fit function is changed I think it should remove the fitted plot.

Reply:
I thought the same, but as far as I could tell the previous behaviour was to keep the old fit on the graph until a new fit was obtained. In an effort to not make any functionality changes in this PR (to keep things small), I tried to replicate the existing behaviour. I can create a new issue to update the plot (i.e. remove the old fit) when the fit function is changed if desired.

Reply:
@StephenSmith25 ok. Leave as is for now